server/controllers/admin/nationalOccupancyController.test.ts

```diff
@@ -136,12 +157,12 @@ describe('NationalOccupancyController', () => {
     })
 
     it('should render the form with data merged from the session and query with query dominant', async () => {
-      const queryData = {
+      const queryData: { arrivalDate: string; apType: ApType } = {
         arrivalDate: '16/12/2025',
-        apType: Object.keys(apTypeLongLabels)[0],
+        apType: makeArrayOfType<ApType>(Object.keys(apTypeLongLabels))[0],
```

Comment:
I have noticed this issue with TypeScript before: surely it should infer the type of the keys as `ApType` for `Object.keys()`, given the object itself is typed as `Record<ApType, string>`? I'm not sure why we have to be more explicit than that. 🤷‍♂️